lib/Caxy/HtmlDiff/AbstractDiff.php

```diff
@@ -154,6 +154,10 @@ public function initPurifier($defaultPurifierSerializerCache = null)
      */
     protected function prepare()
     {
+        if (!$this->config->isPurifierEnabled()) {
```

Comment:
Could you please do strict type checking using tripple `=`, I am slowly removing the `!` operator from this library as it is easy to mis, and PHP Jit performs better if the strict operator is used. (same for the other if statements).